src/main/java/edu/harvard/iq/dataverse/api/Dataverses.java

```diff
@@ -342,15 +343,35 @@ public Response setMetadataRoot( @PathParam("identifier")String dvIdtf, String b
     
     @GET
     @Path("{identifier}/facets/")
+    /**
+     * return list of facets for the dataverse with alias `dvIdtf`
+     */
     public Response listFacets( @PathParam("identifier") String dvIdtf ) {
```

Comment:
I agree that it'd be good to have (I assume it worked once upon a time as well); but the logic of setting up those tests wasn't too clear to me on initial reading (I'm also not sure if those are RestAssured tests that I'm not setup to run locally or not).